influxdb/client.py

```diff
@@ -411,9 +411,10 @@ def write_points(self,
                                       tags=tags, protocol=protocol)
 
     def ping(self):
-        """Check connectivity to influxdb
-        :returns: The version of the influxdb the client is connected to
+        """Check connectivity to InfluxDB
+        :returns: The version of the InfluxDB the client is connected to
         """
+
```

Comment:
Blank line should be after the summary line (i.e. the first line of the docstring), before the description. In PEP 257 no blank lines allowed after the method's docstring. Also, please note that PEP 257 requires that the first line of the docstring ends with the period. 
On the other side, there are currently quite many inconsistencies in the client's docstrings and tox does not check code for PEP 257. May be all these inconsistencies should be revised in a separate PR.